### DISCUSSION TITLE
Style of controls

### DISCUSSION THREAD

**Discussion by Steellgold** [histoire-dev/histoire#275] [Q&A] [answered]
Hey !

I would like to know if it is normal that it is not the same style of boxes in my local story and the demo version on the site 

Local: 
![image](https://user-images.githubusercontent.com/51505384/187537728-502a9bb6-4515-471d-a1ec-8d3a9f9d5c32.png)

Histoire ([Link](https://svelte3.examples.histoire.dev/story/src-sharecontrols-story-svelte?variantId=src-sharecontrols-story-svelte-0&tab=)): 
![image](https://user-images.githubusercontent.com/51505384/187537768-5bd9c4ec-c809-42c7-822f-f0680b1fc8b3.png)

My code:
```HTML
<script lang="ts">
  import Alert from "./alert.svelte";
  export let Hst: any;

  let type: "error" | "success" | "warning" | "info" = "info";
  let message: string = "This is a " + type + " alert";
</script>

<Hst.Story title="Alert">
  <svelte:fragment slot="controls">
    Message: <input type="text" bind:value={message}>
    <br>
    Type: <select bind:value={type}>
      <option value="error">Error</option>
      <option value="success">Success</option>
      <option value="warning">Warning</option>
      <option value="info">Info</option>
    </select>
  </svelte:fragment>

  <Alert type={type} message={message} />
</Hst.Story>
```

**Comment by Akryum** [comment, marked as answer]
You are using HTML controls so it's up to you to style them. If you want, you can also use the builtin controls like in the [examples](https://github.com/histoire-dev/histoire/blob/b57aad827572cc3ac2bff8ce1b955ec61c2bee48/examples/svelte3/src/BaseButton.story.svelte#L23-L31). The list of the controls is [here](https://controls.histoire.dev/).
